queries.sql

```diff
@@ -171,4 +171,10 @@ JOIN(SELECT ANIMALS.SPECIES_ID, COUNT(*) AS SP_COUNT FROM ANIMALS
 	JOIN VETS ON VISITS.VET_ID=VETS.ID
 	WHERE VETS.NAME='Maisy Smith' GROUP BY ANIMALS.SPECIES_ID
 	ORDER BY SP_COUNT DESC LIMIT 1) SP
-ON SP.SPECIES_ID=SPECIES.ID;
\ No newline at end of file
+ON SP.SPECIES_ID=SPECIES.ID;
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4; 
+
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+
```

Comment:
- [ ] In the screenshots for the first two query results the execution time is bigger than 1000ms but it should be smaller then 1000ms as requested in the project requirements. Please make sure to fix this and update the screenshot in the PR description. You can follow the below steps to achieve this.

![image](https://github.com/ZewdieMc/vet-clinic/assets/83803414/1dbf2f2b-258d-416c-a683-0231a7a35f24)

Reply:
Hi @thecodechaser, 
Thank you for your feedback.
Kindly note that the point is not among the project requirements, it's a preparation to proceed to the project requirements and it's saying that the execution time of the `SELECT count(*) from visits where animal_id = 4` should be `>= 1000ms` not `< 1000ms`.
We apologize if we understood it.